CHANGELOG.md

```diff
@@ -43,3 +43,27 @@ Additions to existing modules
   ∸-suc : m ≤ n → suc n ∸ m ≡ suc (n ∸ m)
   ```
 
+* In `Data.Vec.Properties`:
+  ```agda
+  updateAt-take : (xs : Vec A (m + n)) (i : Fin m) (f : A → A) →
+                  updateAt (take m xs) i f ≡ take m (updateAt xs (inject≤ i (m≤m+n m n)) f)
+
+  truncate-zipWith : (m≤n : m ≤ n) (xs : Vec A n) (ys : Vec B n) (f : A → B → C) →
+                    truncate m≤n (zipWith f xs ys) ≡ zipWith f (truncate m≤n xs) (truncate m≤n ys)
```

Comment:
Sorry if I was ambiguous in my review: `updateAt` takes its `f` argument in *final* position, so it makes sense for its properties to mirror that; but `zipWith` takes *its* `f` argument in *first position, so...
```suggestion
  truncate-zipWith : (f : A → B → C) (m≤n : m ≤ n) (xs : Vec A n) (ys : Vec B n) →
                    truncate m≤n (zipWith f xs ys) ≡ zipWith f (truncate m≤n xs) (truncate m≤n ys)
```
or perhaps
```suggestion
  truncate-zipWith : (m≤n : m ≤ n) (f : A → B → C) (xs : Vec A n) (ys : Vec B n) →
                    truncate m≤n (zipWith f xs ys) ≡ zipWith f (truncate m≤n xs) (truncate m≤n ys)
```
if you think `truncate` takes priority here?